pkg/trace/config/config.go

```diff
@@ -267,9 +267,9 @@ type EVPProxy struct {
 	// DDURL is the Datadog site to forward payloads to (defaults to the Site setting if not set).
 	DDURL string
 	// APIKey is the main API Key (defaults to the main API key).
-	APIKey string
+	APIKey string `json:"-"` // Never marshal this field
```

Comment:
Should we do this throughout? e.g. `DebuggerProxyConfig` has an API key as well.